src/Microsoft.TestPlatform.CrossPlatEngine/EventHandlers/IPathConverter.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CommunicationUtilities;
+
+using System.Collections.Generic;
+using ObjectModel;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Client;
+using System.Collections.ObjectModel;
```

Comment:
Nit
```suggestion
using System.Collections.Generic;
using System.Collections.ObjectModel;

using ObjectModel;
using Microsoft.VisualStudio.TestPlatform.ObjectModel;
using Microsoft.VisualStudio.TestPlatform.ObjectModel.Client;
```

Reply:
I am really confused by some of the code style rules that don't seem to always raise.